src/Razor/src/Microsoft.VisualStudio.Editor.Razor/BraceSmartIndenter.cs

```diff
@@ -295,6 +295,7 @@ internal static bool ContainsInvalidContent(SyntaxNode owner)
         return false;
     }
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
This code is only called from the legacy editor, which has much lower usage, so has not been flagged. Sadly this logic is not able to simply be changed to the new method, as it has a different use case.